vg/vgsvg/vgsvg.go

```diff
@@ -461,3 +482,13 @@ func opacityString(clr color.Color) string {
 	_, _, _, a := clr.RGBA()
 	return fmt.Sprintf("%.*g", pr, float64(a)/math.MaxUint16)
 }
+
+// return the class name
```

Comment:
this comment doesn't add too much, and doesn't follow the usual Go code documentation conventions.
could you please add some more context?